CodeReview/styles/item/VCommentWithDateNode/default

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style prototypes="item/VComposite/default" />
```

Comment:
Errr? where is the 'headerBackground' property? Reusing `VComposite`, while technically working, isn't a good idea, since it's more of an accident and it could change at any time. Please define all the properties here, and use DeclarativeItemBase as the prototype.